### PR TITLE
wipe: Add a force flag for skipping version check

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

**Comment by mrunalp** [comment]
@umohnani8 @giuseppe ptal.

**Comment by umohnani8** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM assuming happy tests

File: cmd/crio/wipe.go [hunk 1]
```diff
@@ -18,6 +18,13 @@ var wipeCommand = &cli.Command{
 	Name:   "wipe",
 	Usage:  "wipe CRI-O's container and image storage",
 	Action: crioWipe,
+	Flags: []cli.Flag{
+		&cli.BoolFlag{
+			Name:    "force",
+			Aliases: []string{"f"},
+			Usage:   "force wipe by skipping the version check",
+		},
+	},
```
Comment (saschagrunert) [inline cmd/crio/wipe.go]: The docs have to be re-generated for this via something like `sudo make clean all docs-generation completions`

Comment (mrunalp) [reply cmd/crio/wipe.go]: done, thanks!

**Pull request merged**